Guides/Installation.md

```diff
@@ -9,7 +9,7 @@ The preferred method of installation for `IGListKit` is using [CocoaPods](https:
 In order to use the latest release of the framework, add the following to your `Podfile`:
 
 ```ruby
-pod 'IGListKit', '~> 3.0'
+pod 'IGListKit', '~> 3.1'
```

Comment:
we don't need to update all of these -- only for major versions.

`~> 3.0` will pull in everything up to but not including `4.0`